extras/ams_wrapper/README.md

```diff
@@ -0,0 +1,271 @@
+
+Model includes a configuration file in json format. See an example configuration file [here](../ams_models/vehicle_attributes_model.json)
+
+Below is a sample of such model:
+```json
+{
+   "subtype" : "vehicleClassification",
+   "type" : "classification",
+   "classifications" : [
+      {
+         "attributes" : [
+            {
+               "name" : "color",
+               "confidence" : 0.289313167333603,
+               "value" : "white"
+            },
+            {
+               "confidence" : 0.0435199774801731,
+               "value" : "gray",
+               "name" : "color"
+            },
+            {
+               "confidence" : 0.00365369999781251,
+               "value" : "yellow",
+               "name" : "color"
+            },
+            {
+               "name" : "color",
+               "confidence" : 0.589816808700562,
+               "value" : "red"
+            },
+            {
+               "confidence" : 0.00414687860757113,
+               "value" : "green",
+               "name" : "color"
+            },
+            {
+               "value" : "blue",
+               "confidence" : 0.0417489744722843,
+               "name" : "color"
+            },
+            {
+               "name" : "color",
+               "value" : "black",
+               "confidence" : 0.0278004333376884
+            }
+         ]
+      },
+      {
+         "attributes" : [
+            {
+               "name" : "type",
+               "confidence" : 0.00293471338227391,
+               "value" : "car"
+            },
+            {
+               "name" : "type",
+               "value" : "van",
+               "confidence" : 0.000757944362703711
+            },
+            {
+               "confidence" : 0.996138155460358,
+               "value" : "truck",
+               "name" : "type"
+            },
+            {
+               "name" : "type",
+               "confidence" : 0.000169166218256578,
+               "value" : "bus"
+            }
+         ]
+      }
+   ]
+}
+```
+
+## Deployment and configuration of OpenVINO Inference Server for LVA    
+
+OpenVINO Inference Server for LVA includes two components which require proper configuration.
+* OpenVINO Model Server - serves all models and executes inference operation
+* LVA REST API wrapper - translates LVA API, run pre and post processing operations, communicates with OVMS via localhost and gRPC interface.
+
+OpenVINO Model server requires file `/opt/ams_models/ovms_config.json` which is by default configured
+to use [4 exemplary models](../ams_models/ovms_config.json).
+
+LVA REST API wrapper requires model configuration files describing all enabled models.
+Models config files should be present in `/opt/ams_models/` folder and their name should have the name 
+matched with the model name configured in `ovms_config.json`.
```

Comment:
They don't have to. Configuration files names could be arbitrary as long as they end with `_model.json`.